src/values/fn_value.rs

```diff
@@ -20,7 +20,7 @@ use crate::basic_block::BasicBlock;
 use crate::debug_info::DISubprogram;
 use crate::module::Linkage;
 use crate::support::{to_c_str, LLVMString};
-use crate::types::{FunctionType, PointerType};
+use crate::types::{AnyType, FunctionType, PointerType};
```

Comment:
1. Is this import necessary?
2. Should get rid of `FunctionValue::print_to_string`

Reply:
Yes, this import is necessary.

Note that `cargo clippy` in CI would tell you about unused imports.